src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/hmi/navi_is_ready_request.h

```diff
@@ -72,6 +72,11 @@ class NaviIsReadyRequest : public app_mngr::commands::RequestToHMI,
    **/
   void on_event(const app_mngr::event_engine::Event& event) OVERRIDE;
 
+  /**
+   * @brief onTimeOut from requrst Controller
+   */
+  void onTimeOut() OVERRIDE;
+
```

Comment:
@LitvinenkoIra no need to add the comment for overridden functions - it is enough to have in interface class only